### PR TITLE
[CVP-2636] Temporarily remove Xunit output

### PR DESCRIPTION
**Pull request by nmars**
Use only JSON output while [Xunit output is being fixed](https://github.com/operator-framework/operator-sdk/issues/5457)

### CODE REVIEW

File: ci-operator/step-registry/optional-operators/cvp-common/scorecard/optional-operators-cvp-common-scorecard-commands.sh [hunk 1]
```diff
@@ -102,10 +102,6 @@ if [ -f "${OPERATOR_DIR}/tests/scorecard/config.yaml" ]; then
 
   # Use the json output format for cvp custom scorecard tests
   CUSTOM_SCORECARD_OUTPUT_FORMAT="json"
-  # Use the xunit output format for msp custom scorecard tests
-  if [ "${TEST_MODE}" == "msp" ]; then
-    CUSTOM_SCORECARD_OUTPUT_FORMAT="xunit"
-  fi
```
Comment (lslebodn) [inline ci-operator/step-registry/optional-operators/cvp-common/scorecard/optional-operators-cvp-common-scorecard-commands.sh:108]: Could we make it configurable ? Or will be done it in other story?
Cause `TEST_MODE` and `OUTPUT_FORMAT` should not be tight at all.

Comment (nmars) [reply ci-operator/step-registry/optional-operators/cvp-common/scorecard/optional-operators-cvp-common-scorecard-commands.sh:108]: I agree we should make this configurable. But let's open a new story.

**Pull request merged**